server/src/test/java/org/opensearch/index/store/lockmanager/FileLockInfoTests.java

```diff
@@ -15,10 +15,24 @@
 public class FileLockInfoTests extends OpenSearchTestCase {
```

Comment:
We are missing test for `getFileToLockNameFromLock`, lets add that as well for both v1 an current lock file extension. This why the bug in logic is not caught.

Reply:
added